cmd/openshift-tests/openshift-tests.go

```diff
@@ -111,6 +112,9 @@ func newRunCommand() *cobra.Command {
 		SilenceErrors: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return mirrorToFile(opt, func() error {
+				if !opt.DryRun {
+					fmt.Fprintln(os.Stderr, "openshift-test version:", version.Get().String())
```

Comment:
space between ':' and version.  Then lgtm

Reply:
`openshift-test version: x.y.z`

Reply:
Actually... should be `fmt.Fprintf(os.Stderr, "%s version: %s\n", filepath.Base(os.Args[0]), version.Get().String())`

Reply:
Space is added by `Fprintln`:
```
openshift-test version: v4.1.0+d56e7d7-2472-dirty
```

Reply:
> Spaces are always added between operands and a newline is appended.

Reply:
Fixed to your proposal. Now it must be perfect